kms-message/test/test_kms_util.c

```diff
@@ -59,3 +60,30 @@ char *
 data_to_hex (const uint8_t *buf, size_t len) {
    return hexlify (buf, len);
 }
+
+char *
+replace_all (const char *input, const char *match, const char *replacement)
```

Comment:
Suggest adding not-null assertions for all parameters.

Reply:
Done.